File: Python/0001-1000/001-100/61-70/70.py
class Solution:
    def climbStairs(self, n: int) -> int:
        if n <= 3:
            return n
        n1 = 2
        n2 = 3
        n3 = 3
        for i in range(3, n):
            n3 = n1 + n2
            n1 = n2
            n2 = n3
        return n3

File: Java/0001-1000/001-100/61-70/70.java
class Solution {
    public int climbStairs(int n) {
        if (n <= 3)
            return n;
        int f1 = 2, f2 = 3, f3 = 4;
        for (int i = 4; i <= n; i++) {
            f3 = f1 + f2;
            f1 = f2;
            f2 = f3;
        }
        return f3;
    }
}